sbt-scala-native-bindgen/src/main/scala/org/scalanative/bindgen/sbt/ScalaNativeBindgenPlugin.scala

```diff
@@ -0,0 +1,98 @@
+package org.scalanative.bindgen.sbt
+
+import sbt._
+import sbt.Keys._
+
+import org.scalanative.bindgen.Bindgen
+
+/**
+ * Generate Scala bindings from C headers.
+ *
+ * == Usage ==
+ *
+ * This plugin must be explicitly enabled. To enable it add the following line
+ * to your `.sbt` file:
+ * {{{
+ * enablePlugins(ScalaNativeBindgenPlugin)
+ * }}}
+ *
+ * By default, the plugin reads the configured header file and generates
+ * a single Scala source file in the managed source directory.
+ *
+ * See the [[https://github.com/kornilova-l/scala-native-bindgen/tree/master/sbt-scala-native-bindgen/src/sbt-test/bindgen/generate example project]].
+ *
+ * == Configuration ==
+ *
+ * Keys are defined in [[ScalaNativeBindgenPlugin.autoImport]].
+ *
+ *  - `nativeBindgenHeader`: The C header file to read.
+ *
+ *  - `nativeBindgenPackage`: Package of the enclosing object.
+ *    No package by default.
+ *
+ *  - `name in nativeBindgen`: Name of the enclosing object.
+ *
+ * @example
+ * {{{
+ * nativeBindgenHeader in Compile := file("/usr/include/ctype.h")
+ * nativeBindgenPackage in Compile := Some("org.example.app")
+ * name in (Compile, nativeBindgen) := "ctype"
+ * }}}
+ */
+object ScalaNativeBindgenPlugin extends AutoPlugin {
+
+  object autoImport {
+    val nativeBindgenPath =
+      taskKey[Option[File]]("Path to the scala-native-bindgen executable")
+    val nativeBindgenHeader = taskKey[File]("C header file")
+    val nativeBindgenPackage =
+      settingKey[Option[String]]("Package for the generated code")
+    val nativeBindgenLink =
+      settingKey[Option[String]]("Name of library to be linked")
+    val nativeBindgenExclude = settingKey[Option[String]]("Exclude prefix")
+    val nativeBindgen        = taskKey[File]("Generate GraphQL API code")
```

Comment:
@Jonas, I suppose you forgot to change the description for `nativeBindgen`, I'll change it to "Generate bindings"

Reply:
Good catch :+1: thanks